pom.xml

```diff
@@ -65,7 +65,7 @@
     <hadoop.version>2.10.2</hadoop.version>
     <spark2.version>2.1.3</spark2.version>
     <hydrator.version>2.10.0</hydrator.version>
-    <commons.version>3.8.1</commons.version>
```

Comment:
I didn't see CVE-2025-48924 in b/427399476 could you attach the relevant bug for this?